### ISSUE TITLE
blink-caret-rate CSS Property doesn't work

### ISSUE THREAD

**Issue by JordanMartinez** [TomasMikula/RichTextFX#426]
Styling this property via CSS does nothing to change the default value of 500ms.

**Comment by JordanMartinez** [comment]
Updating `styled-text-area.css` to:
````css
.styled-text-area .virtual-flow {
    -fx-cursor: text;
    -fx-caret-blink-rate: 100ms;
    -fx-font-size: 32pt;
}
````
and this code...
````java
import javafx.application.Application;
import javafx.scene.Scene;
import javafx.stage.Stage;
import org.fxmisc.flowless.VirtualizedScrollPane;
import org.fxmisc.richtext.StyleClassedTextArea;
import org.fxmisc.richtext.StyledTextArea;

import java.util.Collection;
import java.util.Collections;

public class CssTest extends Application {

    public static void main(String[] args) {
        launch(args);
    }

    @Override
    public void start(Stage primaryStage) {
        StyledTextArea<String, Collection<String>> area = new StyledTextArea<String, Collection<String>>("", (a, b) -> {},
                Collections.emptyList(), (a, b) -> a.getStyleClass().addAll(b));
        area.replaceText("Some text in the area");
        VirtualizedScrollPane<StyledTextArea<String, Collection<String>>> vsPane = new VirtualizedScrollPane<>(area);

        Scene scene = new Scene(vsPane, 500, 500);
        scene.getStylesheets().add(StyleClassedTextArea.class.getResource("styled-text-area.css").toExternalForm());
        primaryStage.setScene(scene);
        primaryStage.show();
    }
}
````

**Comment by JFormDesigner** [comment]
@JordanMartinez it works with PR #398 and without `.virtual-flow` in the CSS selector:
```css
.styled-text-area {
    -fx-caret-blink-rate: 100ms;
}
```

**Comment by JordanMartinez** [comment]
Thanks! However, why does that inclusion cause this problem in the first place?

**Comment by JFormDesigner** [comment]
Because `-fx-caret-blink-rate` and `-fx-highlight-fill` CSS properties are not defined in `VirtualFlow` or any of its children. They are defined in class `GenericStyledArea`, which has CSS class `styled-text-area`.

**Comment by JordanMartinez** [comment]
Good to know.

Closed by #398
